spec/unit/dpkg_database_spec.rb

```diff
@@ -68,6 +68,16 @@
     end
   end
 
+  describe "#determine_changes" do
```

Comment:
if the method can be renamed then this could just be a context scenario inside the #parse_changes_line